examples/cloud-armor/README.md

```diff
@@ -0,0 +1,18 @@
+# White listing ip's using Cloud Armor
+
+This is an example of setting up a project to take advatange of one of the [Cloud Armor features](https://cloud.google.com/armor/) that allows whitelisting of traffic to a compute instance based on ip address. It will set up a single compute instance running nginx that is accesible via a load balanced pool that is managed by cloud armor security policies.
+
+To run the example:
+* Set up a gcloud account with the compute engine api enabled
```

Comment:
Unrelated and totally hijacking: What's the best way to get an interactive account working with Terraform w/o using gcloud? Service accounts can't hold some privileges like org owner privileges, and you need those (or some other credentials they can't hold) for some operations like making organization-level logging sinks.

I was only able to test #1860 by using gcloud and couldn't get a service account working (and to my best knowledge can't, but I could be very wrong). What alternative should I have used for an interactive account / is gcloud the best way but we only want it if *really* necessary?

Reply:
We talked about this in person, but in case someone stumbles upon this later: For now, gcloud is the only option. In the future, I'd like to add an oauth flow so that users can use their user account but their credentials come from a project that our team owns instead of one that the gcloud team owns.